speechbrain/lobes/models/kmeans.py

```diff
@@ -1,178 +1,11 @@
-"""K-means implementation.
+"""This file ensures old links to kmeans continue to work while providing a Deprecation warning"""
```

Comment:
this is something I am not fully sure to understand, isn't the `deprecation` module allows to remove files and still have the compatibility of doing let's say `speechbrain.lobes.models.kmeans` ?

Reply:
I just looked again, and I don't think there's any automatic way to do this.